### PR TITLE
newbranch54: Double posting issue

### PR DESCRIPTION
**Pull request by gudmdharalds**

### CODE REVIEW

File: test.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo "test" . time() . "bla" . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+echo "test" . time() . "bla" . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?php
+
+echo "test" . time() . "bla" . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+echo "test" . time() . "bla" . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (gudmdharalds) [inline test.php]: VIP: File is approved.

File: test.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+
+echo "test" . time() . "bla" . PHP_EOL;
```
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 4 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

File: test.php (hunk 6 above)
Comment (gudmdharalds-a8c) [inline test.php]: :no_entry_sign: **Error**: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'time' (*WordPress.Security.EscapeOutput.OutputNotEscaped*).

**Pull request closed without merging**